### ISSUE TITLE
Add maxWidth property

### ISSUE THREAD

**Issue by MHX792** [jdg/MBProgressHUD#551]
A popup should not fill the whole width of the screen. Thanks

**Comment by iPermanent** [comment]
agree, I think it is necessary for some requirements, it has minSize but no maxSize or maxWidth property.

**Comment by DeqiuTseng** [comment]
agree, I think it is necessary for some requirements, it has minSize but no maxSize or maxWidth property.
